backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java

```diff
@@ -95,6 +97,18 @@ public List<BucketResponse> listAllDeletedBuckets() {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public void activateLambda(UUID bucketId, LambdaType lambdaType) {
+    BucketEntity bucketEntity = findBucketEntityById(bucketId);
+    List<LambdaType> activatedLambdas = bucketEntity.getLambdaTypes();
+    if (activatedLambdas.contains(lambdaType)) {
+      throw new LambdaAlreadyActivated();
+    }
+    activatedLambdas.add(lambdaType);
+    BucketEntity bucketWithActivatedLambda = bucketEntity.withLambdaTypes(activatedLambdas);
+    bucketRepository.save(bucketWithActivatedLambda);
```

Comment:
If I am not mistaking if the relation is done ok, the save method is obsolete if you add the lambdaType the way I describe in the previous comment

Reply:
I think so too, yes.

Reply:
did you try it?